org-clock-csv.el

```diff
@@ -243,6 +242,19 @@ for use in batch mode."
             entries))
     (switch-to-buffer buffer)))
 
+;;;###autoload
+(defun org-clock-export-to-csv-file (csv-file &optional infile)
+  "Export clock entries from INFILE to CSV-FILE.
+
+See `org-clock-csv' function for INFILE usage. If INFILE is nil,
+`org-agenda-files' is used as clock source."
+  (interactive "FCSV file: ")
+  (org-clock-csv infile)
+  (with-current-buffer "*clock-entries-csv*"
```

Comment:
Maybe instead of calling this you could use the return value of `(org-clock-csv infile)` directly so that if it changes the name in the future wouldn't have to be updated everywhere.

Obviously then we need to make sure the function always returns the buffer with the data, but that's a good practice observed on many places in Emacs already so there's a strong precedence.